config/settings.yml

```diff
@@ -26,3 +26,6 @@ plugins:
   ldap_password:
     client: false
     default: ''
+  ldap_filter:
+    client: false
+    default: '(&(sAMAccountName=%{username})(memberof=CN=group-example,DC=example,DC=com))'
```

Comment:
I'm agree with you. This will confuse the users... I removed the default example.